### PR TITLE
Update for qt 5.12

### PR DESCRIPTION
**Pull request by jschueller**
@conda-forge-admin please rerender

### CODE REVIEW

**Comment by jschueller** [comment]
ping @conda-forge/qwtpolar

**Comment by jschueller** [comment]
could someone from @conda-forge/core merge this ?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,28 +16,48 @@ source:
...
+    - {{ cdt('libxxf86vm') }}            # [linux]
+    - {{ cdt('libxrandr') }}             # [linux]
+    - {{ cdt('alsa-lib') }}              # [linux]
+    - {{ cdt('xorg-x11-server-xvfb') }}  # [linux]
+    - {{ cdt('pciutils-devel') }}        # [linux]
   host:
     - qt
     - qwt
-  run:
-    - qt
```
Comment (ocefpaf) [inline recipe/meta.yaml:39]: Are you sure about this removals?

File: recipe/yum_requirements.txt [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+libXcursor-devel
+libxcb-devel
+libXau-devel
+libXi-devel
+libXScrnSaver-devel
+libXrandr-devel
+alsa-lib-devel
+xcb-util-devel
+xorg-x11-server-Xvfb
+pciutils-devel
```
Comment (ocefpaf) [inline recipe/yum_requirements.txt]: That is a lot of yum requirements, is it possible to reduce to a minimal working version or do you really need all of those?

File: recipe/meta.yaml (hunk 1 above)
Comment (jschueller) [reply recipe/meta.yaml:39]: yes, both qt and qwt use exports

File: recipe/yum_requirements.txt (hunk 2 above)
Comment (jschueller) [reply recipe/yum_requirements.txt]: @ocefpaf you were right, only the mesa packages are required

**Comment by jschueller** [comment]
@conda-forge-admin please rerender

**Pull request merged**